packages/react-composites/jest.config.js

```diff
@@ -10,8 +10,7 @@ module.exports = {
   ...commonConfig,
   roots: [
     path.join(__dirname, 'src')
-    // Removing the `tests` dir from jest tests directories to prevent these tests
-    // from running with the command `npm test` | `rushx test`
+    // Uncomment the following line to run E2E browser tests
```

Comment:
Can we have an e2e.jest.config instead of having to manipulate this file?

Reply:
Then also a test:e2e command in the package.json that targets this maybe?

Reply:
Done

Reply:
Great advice. Thanks